app/app_test.go

```diff
@@ -69,45 +88,51 @@ func (at *appTest) reset() {
 	require.True(at.t, resabci.IsOK(), resabci)
 }
 
+func getBalance(pk crypto.PubKey, state types.KVStore) (types.Coins, error) {
+	return getAddr(pk.Address(), state)
+}
+
+func getAddr(addr []byte, state types.KVStore) (types.Coins, error) {
+	actor := stack.SigPerm(addr)
+	acct, err := coin.NewAccountant("").GetAccount(state, actor)
```

Comment:
Generally the acronym `acct` for accountant is a bit of a noodler - so close to acc for account, also mixes me up with all the new `Actors`, not sure if there is a better acro' here though... `accnt` may be better for my head